DataHub/Operation/03_ManageHTMLTemplates.md

```diff
@@ -0,0 +1,296 @@
+[!!User interface HTML templates](../UserInterface/02j_HTMLTemplates.md)
+
+# Manage the HTML templates
+
+Define HTML templates that are used for the ETL mapping, especially for the receipt export.
+HTML templates allow you to include HTML text or Smarty templates and thus create complex queries with many logical inputs.
```

Comment:
nicht glücklich mit dem satz.
eher irgendwie in die richtung dass einem die html templates mit hilfe der smarty-template-engine erlauben html-inhalte aus daten an hand von vorlagen zu generieren. 

irgendwas in die richtung